Model/Smartcalcs.php

```diff
@@ -202,15 +209,27 @@ public function getTaxForOrder(
             $client->setUri(TaxjarConfig::TAXJAR_API_URL . '/magento/taxes');
             $client->setHeaders('Authorization', 'Bearer ' . $apiKey);
             $client->setRawData(json_encode($order), 'application/json');
+            $this->logger->log('Calculating sales tax: ' . json_encode($order), 'post');
 
             $this->_setSessionData('order', json_encode($order));
 
             try {
                 $response = $client->request('POST');
                 $this->response = $response;
                 $this->_setSessionData('response', $response);
+
+                if ($response->getStatus() ==200 ) // Success
```

Comment:
Space between `==` and `200`, braces on same line for conditionals similar to M1